pkg/controller/servicemesh/controlplane/readiness.go

```diff
@@ -175,8 +177,16 @@ func (r *controlPlaneInstanceReconciler) calculateComponentReadinessMap(ctx cont
 			},
 		},
 	}
+
+	namespaces, err := r.getNamespacesToCheck()
+	if err != nil {
+		return nil, err
+	}
+
+	log.V(2).Info("Calculating readiness", "namespaces", namespaces)
```

Comment:
Can we add a component name here?

Reply:
What do you mean by "component"?

Reply:
I mean Deployment, StatefulSet, and DaemonSet. Move this log to the `typesToCheck` loop and just print for which type it is calculating readiness. Not sure if that is going to be super useful.

Reply:
I don't think we need to log at the lower abstraction layer than we currently are, because it doesn't really matter what the resource type is. I actually added this line because I needed to see which namespaces it was scanning. Also, we do log each resource that we find (at V(3)), so if someone needs to see additional details of the scan, they can just turn that on.

Reply:
Sounds good.